docs/setting-up/server/helm.md

```diff
@@ -13,32 +10,84 @@
 !!! note alert alert-primary ""
     Helm v3 is needed to run the following steps.
 
+Refer to [Kubernetes Supported versions](https://kubernetes.io/releases/version-skew-policy/#supported-versions) and [Helm Version Support Policy](https://helm.sh/docs/topics/version_skew/) to find the supported versions.
+
+PMM should be platform-agnostic, but it requires escalated privileges inside a container. It is necessary to have a `root` user inside the PMM container. Thus, PMM would not work for Kubernetes Platforms such as OpenShift or others that have hardened Security Context Constraints, for example:
+
+- [Security context constraints (SCCs)
+](https://docs.openshift.com/container-platform/latest/security/container_security/security-platform.html#security-deployment-sccs_security-platform)
+- [Managing security context constraints](https://docs.openshift.com/container-platform/latest/authentication/managing-security-context-constraints.html)
+
+Kubernetes platforms offer a different set of capabilities. To use PMM in production, you would need backups and, thus storage driver that supports snapshots. Consult your provider for Kubernetes and Cloud storage capabilities.
+
+## Locality and Availability
+
+You should not run the PMM monitoring server along with the monitored database clusters and services on the same system.
+
+Please ensure proper locality either by physically separating workloads in Kubernetes clusters or running separate Kubernetes clusters for the databases and monitoring workloads.
+
+You can physically separate workloads by properly configuring Kubernetes nodes, affinity rules, label selections, etc.
+
+Also, ensure that the Kubernetes cluster has [high availability](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/ha-topology/) so that in case of a node failure, the monitoring service will be running and capturing the required data.
+
+When configured correctly, PMM, along with Kubernetes' high availability, would be highly available. Ensure that your network and storage support high availability. In the event of a Kubernetes node failure, PMM will be scheduled on any other node in the Kubernetes cluster.
+
```

Comment:
@denisok why dont we use [this paragraph](https://docs.google.com/document/d/1A3t8pYciPB8xyeD_iZhl2mNZtI-H704xp5yYOSU0k9U/edit#bookmark=id.gq45i8o4hzxw)

Reply:
I would vote to remove this HA section at all to not confuse ppl.